pagopa/pn-delivery-push
# fix: add send logo to legal facts and edit for first paragraph

### Description
Adds send logo and description to all legal facts

Involves:
- PdfFileCompliance.html
- PecDeliveryWorkflowLegalFact.html
- NotificationViewedLegalFact.html
- NotificationReceivedLegalFact.html
- AnalogDeliveryWorkflowFailureLegalFact.html

#### How to test
Generate legal facts and check that on the top right of the file the SEND logo is present, in the first paragraph the sentence "(anche nota come SEND - Servizio Notifiche Digitali)" should be present.
